packages/SwingSet/src/kernel/kernel.js

```diff
@@ -607,6 +638,10 @@ export default function buildKernel(kernelEndowments) {
     // the VatManager+VatWorker will see the error case, but liveslots will
     // not
     function vatSyscallHandler(vatSyscallObject) {
+      if (ephemeral.vats.get(vatID).dead) {
+        console.error(`vatSyscallHandler invoked on dead vat ${vatID}`);
```

Comment:
let's add a comment that this is guarding against a confused vatManager that allows the supposedly-dead vat to keep making syscalls, just for the benefit of ourselves reading this code later and puzzling over how this could possibly be reached